packages/mobile/src/account/Profile.tsx

```diff
@@ -19,9 +19,8 @@ interface StateProps {
   userContact: UserContactDetails
 }
 
-interface OwnProps {
-  navigation: any
-}
+// tslint:disable-next-line: no-empty-interface
+interface OwnProps {}
```

Comment:
Why not just remove OwnProps from here and the next line?

Reply:
Right! Left this in case we wanted to add some props again. This screen will get refactored with the redesign and this will probably go away too.